docs/windows.md

```diff
@@ -0,0 +1,210 @@
+# Antrea on Windows
+
+## Overview
+Antrea supports Windows worker Node. On Windows Node, Antrea sets up an overlay network to forward packets between nodes.
+Currently VXLAN and STT tunnels are supported.
+
+This page shows how to install Antrea Agent on Windows Nodes and register the Node to existing kubernetes cluster.
+
+For the detailed design of how Antrea Agent works on Windows, please refer to the doc [Antrea on Windows](
+https://docs.google.com/document/d/1lSis0XnKz8UcJSkxTgRtDhP2DAwQtcZDjT6ZRySUb48)
+
+### Components that run on Windows
+
+The following components should be configured and run on the Windows Node.
+* [kubernetes components](https://kubernetes.io/docs/setup/production-environment/windows/user-guide-windows-nodes/)
+* OVS daemons
+* Antrea Agent
+* Kube-proxy
+
+Antrea-Agent and Kube-proxy run as processes on host and are managed by management Pods. It is recommended to run OVS
+daemons as Windows services.
+
+## Deploying Antrea on Windows Worker Node
+
+### Prerequisite
+* Obtain a Windows Server 2019 license (or higher) in order to configure the
+Windows Node that hosts Windows containers. And install the latest Windows updates.
+* Deploy a Linux-based Kubernetes cluster.
+* Install [Hyper-V](https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/get-started/install-the-hyper-v-role-on-windows-server) with management tools
```

Comment:
To keep consistency, please add an ending period for all list items or none of them.

Reply:
Done